### ISSUE TITLE
[SF_ACC][MultiInput]: Screen reader does not announce value help icon

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#10443]
### Bug Description

When navigating to the input field, ensure that the screen reader announces the presence of the value help icon, making its functionality clear to users relying on assistive technologies. 
Ensure that all elements visible to sighted users are equally accessible to screen reader users. 
[Refer UI5 for implementation](https://ui5.sap.com/#/entity/sap.m.MultiInput/sample/sap.m.sample.MultiInputValueHelp)
![Image](https://github.com/user-attachments/assets/0effc59b-caf4-4519-bdee-458fa3d9a95f)

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by ivoplashkov** [comment]
Hello colleagues,

Seems like value help is not announced when user focuses the multiinput and this is not the case in UI5. - [Isolated example](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cbjxoZWFkPlxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpOyBoZWlnaHQ6IDM1MHB4XCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1tdWx0aS1pbnB1dCBpZD1cIm11bHRpLWlucHV0XCIgc2hvdy12YWx1ZS1oZWxwLWljb24-XG4gICAgICAgIDx1aTUtdG9rZW4gc2xvdD1cInRva2Vuc1wiIHRleHQ9XCJBcmdlbnRpbmFcIj48L3VpNS10b2tlbj5cbiAgICAgICAgPHVpNS10b2tlbiBzbG90PVwidG9rZW5zXCIgdGV4dD1cIk1leGljb1wiPjwvdWk1LXRva2VuPlxuICAgICAgICA8dWk1LXRva2VuIHNsb3Q9XCJ0b2tlbnNcIiB0ZXh0PVwiUGhpbGlwcGluZXNcIj48L3VpNS10b2tlbj5cbiAgICAgICAgPHVpNS10b2tlbiBzbG90PVwidG9rZW5zXCIgdGV4dD1cIlN3ZWRlblwiPjwvdWk1LXRva2VuPlxuICAgICAgICA8dWk1LXRva2VuIHNsb3Q9XCJ0b2tlbnNcIiB0ZXh0PVwiVVNBXCI-PC91aTUtdG9rZW4-XG4gICAgPC91aTUtbXVsdGktaW5wdXQ-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9NdWx0aUlucHV0LmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9Ub2tlbi5qc1wiO1xuXG5kb2N1bWVudC5nZXRFbGVtZW50QnlJZChcIm11bHRpLWlucHV0XCIpLmFkZEV2ZW50TGlzdGVuZXIoXCJ0b2tlbi1kZWxldGVcIiwgZnVuY3Rpb24gKGV2ZW50KSB7XG5cdGNvbnN0IHRva2VucyA9IGV2ZW50LmRldGFpbD8udG9rZW5zO1xuXG5cdGlmICh0b2tlbnMpIHtcblx0XHR0b2tlbnMuZm9yRWFjaCh0b2tlbiA9PiB0b2tlbi5yZW1vdmUoKSk7XG5cdH1cbn0pOyJ9fQ)

Could you please have a look?
Thanks and regards,
Ivaylo
